src/node_http2.cc

```diff
@@ -1700,6 +1700,13 @@ void Http2Session::OnStreamRead(ssize_t nread, const uv_buf_t& buf) {
   stream_buf_ = uv_buf_init(nullptr, 0);
 }
 
+bool Http2Session::HasWritesOnSocketForStream(Http2Stream* stream) {
+  for (const nghttp2_stream_write& wr : outgoing_buffers_)
```

Comment:
Nit: can we put brackets around this multi-line block?

Reply:
@apapirovski Sure, done. We’re not that strict with that in the C++ parts usually

Reply:
I would prefer we were, haha. It's just easier to read :) Thanks!